config/environment.js

```diff
@@ -1,5 +1,15 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return { };
+  /*
+    Have to provide ember-resize defaults due to https://github.com/mike-north/ember-resize/issues/23
+  */
+  return {
+    resizeServiceDefaults: {
+      debounceTimeout    : 200,
+      heightSensitive    : true,
+      widthSensitive     : true,
+      injectionFactories : [ 'view', 'component'],
```

Comment:
I've been running this on Ember 2.5.x and have not seen any warnings. I also don't see any [ember-resize issues](https://github.com/mike-north/ember-resize/issues?q=is%3Aopen+is%3Aissue) about it.